lib/resource/ApiKey.js

```diff
@@ -84,4 +83,9 @@ ApiKey.prototype._decrypt = function _decrypt(callback) {
   }
 };
 
+ApiKey.prototype.getAccount = function () {
```

Comment:
This is missing a JSDoc block comment. Please see `AuthenticationResult.getAccount()` as a reference to the comment that you need to add:

```
/**
 * Retrieves the account resource of the user that has been authenticated.
 *
 * @param {ExpansionOptions} [options]
 * For retrieving linked resources of the {@link Account} during this request.
 *
 * @param {Function} Callback
 * Callback function, will be called with (err, {@link Account account}).
 */
```